### DISCUSSION TITLE
Hello respected community, I have a problem [ASUS PRIME X570-P - Nvidia RTX2070 Super] ~ Hola respetada comunidad, tengo un problema [ASUS PRIME X570-P - Nvidia RTX2070 Super]

### DISCUSSION THREAD

**Discussion by LitohS76** [terminatorul/NvStrapsReBar#86] [Q&A]
Hello everyone // Hola a todos!
I'm looking for help to enable ReBar on my PC // Busco ayuda para habilitar ReBar en mi pc.
  ════════════════════════════════════════════════════

<details><summary>English</summary>
<p>

What happens is that I was trying to enable the Resizable BAR on my Nvidia RTX 2070 Super graphics card, but I haven't had any luck with that, my components are:

CPU: Ryzen 5 3600
Motherboard: ASUS PRIME X570-P
Graphics: Nvidia RTX 2070 Super
RAM: 16GB

<details><summary>IMG</summary>
<p>

![CPUZ GPUZ](https://github.com/terminatorul/NvStrapsReBar/assets/151583746/cd5cd647-9deb-498c-a30e-ea974c6d6702)

</p>
</details> 

I am a user who has no experience modifying BIOS of any kind, so I have had problems trying to add the NvStrapsReBar.ffs file to the BIOS of my Motherboard, but I wanted to ask you since you are the real experts in this field, if Could someone teach me via Discord or tell me why the BIOS cannot be patched, maybe it's me misunderstanding the procedure, but I can't enable it in any way, I've been watching this video and followed all the steps until I got the files. :

NvStrapsReBar.exe
NvStrapsReBar.ffs

As shown in the video, https://www.youtube.com/watch?v=ldnC18u8FI0
but then when using UEFItool.exe it appears different to how it appears in the video since when searching I see two codes instead of one as I show in the following image:

<details><summary>IMG</summary>
<p>

![Search and do a Header only GUID](https://github.com/terminatorul/NvStrapsReBar/assets/151583746/e0ce5353-0a34-4476-8540-82ff68eb0aba)

</p>
</details> 

So following the procedure seems confusing to me, when I try to add the module and save it doesn't give me any problems, but when I do the step of using UEFIPatch.exe it gives me an error and nothing happens as I show in the following image:

<details><summary>IMG</summary>
<p>

![Patch BIOS NvStrapsReBar](https://github.com/terminatorul/NvStrapsReBar/assets/151583746/87e7d34d-7a6a-4f6d-8465-898a5ff4aeff)

</p>
</details> 

It is worth mentioning that my Motherboard does support ReBar and I understand that the only thing left to do is add NvStrapsReBar.ffs that I created following the instructions in the video I mentioned above to the BIOS, but I don't know what else to do or how to enable it, that's why I created this new discussion, I will leave the files in case someone dares to try and help me:

Discord: sensei.killz

Thank you very much for your help!

 </p>
</details> 

</p>
</details> 

  ════════════════════════════════════════════════════

<details><summary>Español</summary>
<p>

Lo que ocurre es que estaba intentando habilitar el Resizable BAR en mi tarjeta grafica Nvidia RTX 2070 Super, pero no eh tenido suerte en eso, mis componentes son:

CPU : Ryzen 5 3600
Motherboard :  ASUS PRIME X570-P
Grafica : Nvidia RTX 2070 Super
RAM : 16GB

<details><summary>IMG</summary>
<p>

![CPUZ GPUZ](https://github.com/terminatorul/NvStrapsReBar/assets/151583746/cd5cd647-9deb-498c-a30e-ea974c6d6702)

</p>
</details> 

Soy un usuario que no tiene experiencia modificando BIOS de ninguna clase, por lo que eh tenido problemas al intentar agregar el archivo NvStrapsReBar.ffs a la BIOS de mi Motherboard, pero quería preguntarles a ustedes ya que son los realmente expertos en este campo, si es que alguien podía enseñarme via Discord o decirme el porque no se puede parchar la BIOS, quizás soy yo que estoy entendiendo mal el procedimiento, pero no consigo habilitarlo de ninguna manera, eh estado viendo este video y seguí todo los pasos hasta obtener los archivos:

NvStrapsReBar.exe
NvStrapsReBar.ffs

Como muestra en el video, https://www.youtube.com/watch?v=ldnC18u8FI0
pero luego al usar el UEFItool.exe me aparece distinto a como le aparece en el video ya que al buscar me aparecen dos códigos en ves de uno como muestro en la siguiente imagen: 
 
<details><summary>IMG</summary>
<p>

![Search and do a Header only GUID](https://github.com/terminatorul/NvStrapsReBar/assets/151583746/e0ce5353-0a34-4476-8540-82ff68eb0aba)

</p>
</details> 

Por lo que seguir el procedimiento se me hace confuso, al intentar agregar el modulo y guardar no me da problemas, pero al momento de hacer el paso de usar UEFIPatch.exe me da un error y no ocurre nada como muestro en la siguiente imagen:

<details><summary>IMG</summary>
<p>

![Patch BIOS NvStrapsReBar](https://github.com/terminatorul/NvStrapsReBar/assets/151583746/87e7d34d-7a6a-4f6d-8465-898a5ff4aeff)

</p>
</details> 

cabe mencionar que mi Motherboard si soporta ReBar y tengo entendido de que solo faltaría agregar NvStrapsReBar.ffs que cree siguiendo las instrucciones del video que mencione mas arriba a la BIOS, pero no se que mas hacer ni como lograr Habilitarlo, por eso creo esta new discussion, les dejare los archivos por si alguien se atreve a intentarlo y me logra ayudar:

Discord: sensei.killz

Muchas gracias por su ayuda! 

 </p>
</details> 

  ════════════════════════════════════════════════════

<details><summary>↓↓↓ Files / Archivos ↓↓↓</summary>
<p>

BIOS PRIME-X570-P-ASUS-5013
NvStrapsReBar.exe
NvStrapsReBar.ffs
MMTOOL 4.50.0.23.zip
UEFIPatch_0.28.0_win32.zip
UEFITool_0.28.0_win32.zip

[PRIME x570-P - 2070 Super.zip](https://github.com/terminatorul/NvStrapsReBar/files/15458756/PRIME.x570-P.-.2070.Super.zip)

<details><summary>IMG</summary>
<p>

![Archivos](https://github.com/terminatorul/NvStrapsReBar/assets/151583746/715cb367-b49e-478d-802d-28d37c94828f)

 </p>
</details> 

 </p>
</details> 

  ════════════════════════════════════════════════════
pd: I apologize for using Google Translate for this entire post. // pido disculpas por que utilice traductor de google para todo este post.

**Comment by terminatorul** [comment]
An X570 motherboard (like yours) is recent enough that I do not think you need UEFIPatch. I think you can just skip it.

The double message in UEFITool  appears to be just a duplicate to me, if you click on it, it will show up (auto-select) the same location for both messages. Just pick the second message and go with it

**Reply by LitohS76** [reply]
Okay, I'm skipping the step, but my question is this: Do I have to add "NvStrapsReBar.ffs" to the BIOS "PRIME-X570-P-ASUS-5013.cap"? What happens is that my BIOS is updated to the latest version, what is that or will I have to Downgrade and then do it?could you help me do it?

**Reply by LitohS76** [reply]
Try to do it as you told me, but the BIOS cannot run, I Downgraded the BIOS thinking that could be it, but this still appears when I try:

<details><summary>Español</summary>
<p>
intente hacerlo como tu me indicaste, pero la BIOS no logra ejecutarse, le hice un Downgrade a la BIOS pensando que podría ser eso, pero igual me aparece esto al intentar:
</p>
</details> 


![WhatsApp Image 2024-05-27 at 23 41 15](https://github.com/terminatorul/NvStrapsReBar/assets/151583746/ca87c3c0-e62d-48f2-a948-c664720c6010)
